### ISSUE TITLE
Project costume looks different in 3.0

### ISSUE THREAD

**Issue by paulkaplan** [LLK/scratch-gui#4127]
Project ID 244605948
- click green flag, then click the right arrow once 
- see the costume is all white in 2.0, but has blue bottom in 3.0

![image](https://user-images.githubusercontent.com/654102/50166122-5da56500-02b4-11e9-8798-9d3dbaacd877.png)

compatibility view: https://llk.github.io/scratch-gui/develop/compatibility-testing.html#244605948

**Comment by ktbee** [comment]
Hmm, I'm wondering if this issue has been fixed @paulkaplan? From what I'm seeing when I load this project in Scratch 3, the suspect astronaut sprite has a blue bottom for just a moment and then is all grey. I found that this flash of blue and then all grey costume is consistent with Scratch 2.

![Screen Shot 2019-03-18 at 4 04 45 PM](https://user-images.githubusercontent.com/7991694/54559959-9b8b5580-4997-11e9-830f-5d8341fcbe92.png)

Let me know if I'm missing some repro steps. This is the project link I'm looking at: https://scratch.mit.edu/projects/244605948
